### PR TITLE
Update the payload of the media to comply with form-data structure

### PR DESCRIPTION
**Pull request by zaher-hs**
Updated the payload as the parser plug will transform it. 
Updated the docs.
Updated the tests.

### CODE REVIEW

File: mix.exs [hunk 1]
```diff
@@ -60,6 +60,7 @@ defmodule Media.MixProject do
       {:plug_cowboy, "~> 2.0"},
       {:ex_aws, "~> 2.1"},
       {:ex_aws_s3, "~> 2.0"},
+      {:sweet_xml, ">= 0.0.0"},
```
Comment (jfayad) [inline mix.exs:63]: @zaher-hs what is this deps for ?

File: Medias.postman_collection.json [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+						},
+						{
+							"key": "files[1][platform_id]",
+							"value": "2",
+							"type": "text"
+						},
+						{
+							"key": "files[1][file]",
+							"type": "file",
+							"src": "/Users/z.hassan/Downloads/Frame 2.png"
```
Comment (jfayad) [inline Medias.postman_collection.json:29]: how is this PATH supposed to work for someone trying this collection ?

File: mix.exs (hunk 1 above)
Comment (zaher-hs) [reply mix.exs:63]: this is used for AWS sending/receiving requests I think it is something related to encoding and decoding the request.

File: Medias.postman_collection.json (hunk 2 above)
Comment (zaher-hs) [reply Medias.postman_collection.json:29]: He will need to update with a file from his file system. I can't put an env variable as a file.

**Review by jfayad — APPROVED** [review]

**Pull request merged**